dev-packages/e2e-tests/test-applications/nuxt-3-dynamic-import/nuxt.config.ts

```diff
@@ -0,0 +1,23 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+  modules: ['@sentry/nuxt/module'],
+  imports: {
+    autoImport: false,
+  },
+  runtimeConfig: {
+    public: {
+      sentry: {
+        dsn: 'https://public@dsn.ingest.sentry.io/1337',
+      },
+    },
+  },
+  nitro: {
+    rollupConfig: {
+      // @sentry/... is set external to prevent bundling all of Sentry into the `runtime.mjs` file in the build output
+      external: [/@sentry\/.*/],
+    },
+  },
+  sentry: {
+    autoInjectServerSentry: 'top-level-import',
```

Comment:
l: If this is means we use the top-level import instead of the dynamic import, can we change the e2e test app name?

Reply:
Actually, there's the other app which does that. Is this option set correctly for the dynamic import?

Reply:
good catch, I need to change the option.